cmd/spec/path.yaml

```diff
@@ -289,3 +289,23 @@ paths:
                 $ref: '#/components/schemas/v1.NoopReservationResponse'
         "500":
           $ref: '#/components/responses/InternalError'
+  /availability_status:
+    post:
+      operationId: availabilityStatus
```

Comment:
Could we include the `Sources` name in the enpoint name?
At this point it is clear what we mean, but later on we can have more availability checks :)

Reply:
Added.